src/router/Routes.tsx

```diff
@@ -31,6 +32,7 @@ const Routes = () => (
         <PrivateRoute path='/module/:id/' component={PublicModule}/>
         <Route exact={true} path={RoutePaths.userModule} component={UserModulePage}/>
         <Route exact={true} path={RoutePaths.logout} component={LogOut}/>
+        <Route exact={true} path={RoutePaths.lab} component={Labs} />
```

Comment:
Labs or Lab? Which one is it? You need to be thinking about singular vs plural in everything you make.